Islandora-Devops/isle-dc
# LAGS-145 - Add Make command to optimize Solr

Ticket: https://jhulibraries.atlassian.net/browse/LAGS-145

## To test
- Go to http://solr-idc.traefik.me/solr/#/ > Core Admin and check if it's not current. If there's a green checkbox you may need to add objects to islandora or run a migrations.
- Run `make solr-reload-cores` and go back to http://solr-idc.traefik.me/solr/#/ > Core Admin and check if the  status of "current" is green. If green this PR worked.
